examples/power_spectra_example.py

```diff
@@ -17,23 +17,26 @@
 from sunpy.data.sample import GOES_XRS_TIMESERIES
 
 ###############################################################################
-# Let's first load a TimeSeries from sample data
+# Let's first load a TimeSeries from sample data.
 
 ts = sunpy.timeseries.TimeSeries(GOES_XRS_TIMESERIES)
+# Data is taken from GOES X-Ray Sensor(XRS), which observes full-disk-integrated
+# solar flux in two broadband channels: 1--8 angstrom (long); and
+# 0.5--4 angstrom (short)
 
 ###############################################################################
 # We now use scipy's periodogram to estimate the power spectra of the first 
 # column of the Timeseries
 
-cols=ts.columns
-freq, spectra = signal.periodogram(ts.data[cols[0]], fs=1)
+xray_short = ts.columns[0]
+freq, spectra = signal.periodogram(ts.data[xray_short], fs=1)
```

Comment:
I've looked in detail into the data and I don't believe this is correct. `fs=1` means the cadence of the data in Hz, the data seems to not be completely evenly sampled!!
```python
>>> sample = np.array([(x - y).total_seconds() for x, y in zip(ts.index[1:], ts.index)])
>>> sample.min(), sample.max(), sample.mean(), sample.std()
  (2.0430000000000001, 14.337000000000002, 2.0485031771623676, 0.077912702228747382)
```
I don't know how there's a 14.33 seconds gap!!! If you plot that you see it happens twice! This means that this analysis would be wrong all together!

In case the cadence would be 2.043, then I think the `fs` should be `1/2.043 s = 0.48 Hz`

Reply:
Should I clean the data by removing those 2 outliers ?

Reply:
Should I just change `fs` to 0.48 and notify the users about the irregularities in data or should I change the input data ?